composer.json

```diff
@@ -16,6 +16,9 @@
         }
     },
     "require": {
-        "phpspec/phpspec": "~2.0"
+        "phpspec/phpspec": "~3.0.0"
```

Comment:
It's because they're referring to an `@internal` API that I promised Matt I won't break in a patch